ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -2032,6 +2032,23 @@ tests:
       WORKFLOW_TYPE: cucushift
       YQ_VERSION: 4.30.5
     workflow: openshift-e2e-ibmcloud-heterogeneous-test-qe
+- as: baremetal-upi-ovn-ipv4-disconnected-proxy-arm-p1-f14
```

Comment:
`-pN (n=1,2,3,...)` is no longer needed.

Reply:
Not sure I got what need to fix here

Reply:
We no long need `-p1` in the test name.